### PR TITLE
Update meta.yaml

### PR DESCRIPTION
**Pull request by DamienIrving**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by DamienIrving** [comment]
@conda-forge-admin please rerender

**Comment by DamienIrving** [comment]
@znicholls - Any idea why we're failing some of the checks here?

File: recipe/meta.yaml
Comment (znicholls) [inline recipe/meta.yaml:1]: It's not clear why, but the sha on pypi has changed (see https://pypi.org/project/cmdline-provenance/#copy-hash-modal-f9bd2fe1-f25d-48c1-9e38-b6d1b6802b5c). 

So the line

```
sha256: a71497e673518c527e6e97a5a845a7339b18d405e4698e979428cc2066e3c5e6
```

needs to become

```
sha256: 3da89243cd315ed2d7929887ebcb82ee00c3a288e00b80e227b8ec51174ed868
```

Comment (DamienIrving) [reply recipe/meta.yaml:1]: That fixed it! Thanks, @znicholls

**Pull request merged**